app/assets/javascripts/discourse/app/templates/preferences-second-factor.hbs

```diff
@@ -190,7 +190,7 @@
 
               {{#unless this.showEnforcedNotice}}
                 <CancelLink
-                  @route="preferences.account"
+                  @route="preferences.security"
```

Comment:
That was a small UX bug (link to 2FA is on the security page but "Cancel" led to the account page)